.github/workflows/check-paper-integrity.yml

```diff
@@ -0,0 +1,21 @@
+name: Test Paper Architecture integrity
+on: [pull_request]
+jobs:
+  check:
+    if: github.repository == 'software-mansion/react-native-gesture-handler'
+    runs-on: ubuntu-latest
+    concurrency:
+      group: static-root-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 18
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - name: Install node dependencies
```

Comment:
You also need to install FabricExample deps to be able to run codegen there

Reply:
Now? https://github.com/software-mansion/react-native-gesture-handler/pull/2933/commits/9b97bf45606026b849c6d831eb83e2e9a2c496ff 🥺